docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
These changes adding `|-` to the `?` runes are inconsistent, but I don't think they're worse than what was there before. I puzzled over those expansions for a while, trying to figure out if `$` could actually recurse into a conditional ...

But maybe that's just me ;)